Base/Python/slicer/tests/test_slicer_parameter_node_wrapper.py

```diff
@@ -0,0 +1,510 @@
+import dataclasses
+import pathlib
+from typing import Annotated
+import unittest
+
+import vtk
+import slicer
+
+from _AverageMesh.TypedParameterNodeMk3 import (
```

Comment:
Should be `from slicer.parameterNodeWraper import (`

Reply:
Good catch, yes.

Reply:
See https://github.com/Slicer/Slicer/pull/6684/commits/279415ebbfc582816e07a53c458918a33ff07002